packages/components/inputs/async-select-input/src/async-select-input.tsx

```diff
@@ -347,8 +364,20 @@ const AsyncSelectInput = (props: TAsyncSelectInputProps) => {
     componentName: 'AsyncSelectInput',
   });
 
-  const placeholder =
-    props.placeholder || intl.formatMessage(messages.placeholder);
+  // const placeholder =
+  //   props.placeholder || intl.formatMessage(messages.placeholder);
+
+  const placeholder = props.placeholder
+    ? props.placeholder
+    : props.appearance === 'filter'
+    ? 'Search'
+    : intl.formatMessage(messages.placeholder);
+
+  // TODO: uncomment to replace placeholder logic once select-input PR for this has been merged.
+  // const placeholder =
+  //   props.appearance === 'filter' && !props.placeholder
+  //     ? intl.formatMessage(messages.selectInputAsFilterPlaceholder)
+  //     : props.placeholder || intl.formatMessage(messages.placeholder);
```

Comment:
You can simplify this condition. Only if there is no props.placeholder you need to check which default-placeholder to pick:

```
const placeholder = props.placeholder || intl.formatMessage(props.appearance === 'filter' ? messages.selectInputAsFilterPlaceholder ? : messages.placeholder);
```

Reply:
updated here: https://github.com/commercetools/ui-kit/pull/2980/commits/66e84c2e6b4acd52c9a5c6df98b208bdcf190f1a
Thank you